.github/workflows/release.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Set up project.
         run: npm run bootstrap
       - name: Run tests.
-        run: USE_REACT_STRICT_MODE=0 npm run test && npm run test:vitest && npm run test:jest && npm run cy:component
+        run: USE_REACT_STRICT_MODE=0 npm run test && npm run test:vitest && npm run cy:component
```

Comment:
we still have 1 `Jest` test suite in the `ui` package, but that will be converted to `vitest` next, so its OK that those tests are not running for a few builds.